test/test_helper.bash

```diff
@@ -16,3 +16,12 @@ setup_script_dir() {
   mkdir -p "${SCRIPT_DIR}"
   cp -a "${src_dirname}/"* "${SCRIPT_DIR}"
 }
+
+cleanup_stubs() {
+  if stat ${BATS_TMPDIR}/*-stub-plan >/dev/null 2>&1; then
```

Comment:
Do we only need to delete the *-stub-plan files, or also the *-stub-run files? Also, do we have to delete the files in the bin folder?

Reply:
Deleting those files should be handled by calling unstub on line 24

Reply:
I see. Looks good!